cmd/oras/internal/option/confirmation.go

```diff
@@ -45,7 +45,12 @@ func (opts *Confirmation) AskForConfirmation(r io.Reader, prompt string) (bool,
 
 	scanner := bufio.NewScanner(r)
 	if ok := scanner.Scan(); !ok {
-		return false, scanner.Err()
+		err := scanner.Err()
+		if err == nil {
+			// EOF
+			fmt.Println("Operation cancelled.")
+		}
+		return false, err
 	}
 	response := scanner.Text()
```

Comment:
How about following the examples in the doc of [bufio.Scanner](https://pkg.go.dev/bufio#Scanner)?
```go
var response string
scanner := bufio.NewScanner(r)
if ok := scanner.Scan(); ok {
	response := scanner.Text()
}
if err := scanner.Err(); err != nil {
	return false, err
}
```

Reply:
Let me check if scanning EOF breaks `strings.ToLower(response)`. If not, this is is definitely better.

Reply:
updated